examples/kitty/Makefile

```diff
@@ -284,6 +284,10 @@ directories:
 	$(shell mkdir -p $(BUILD_DIR)/lwip/src/core/ipv4)
 	$(shell mkdir -p $(BUILD_DIR)/nfs)
 
+submodules:
+	git submodule update --init
```

Comment:
This will initialise every submodule we have, whereas we want to only initialise the ones we need.

Reply:
(right now I think that's everything in `dep/` except `microdot`.

Reply:
I was following what was in the LionsOS documentation, which does `git submodule update --init`. I'll reduce down to just the ones we need.

Reply:
Ah okay, yea that documentation is wrong then.

I guess until recently all the submodules were dependencies of Kitty but not anymore.